thirdparty/libunibreak/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+project(libunibreak)
+cmake_minimum_required(VERSION 3.5.1)
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/../cmake_modules")
+include("koreader_thirdparty_common")
+include("koreader_thirdparty_git")
+
+enable_language(C)
+
+assert_var_defined(CC)
+assert_var_defined(CFLAGS)
+assert_var_defined(CHOST)
+assert_var_defined(LDFLAGS)
+
+ep_get_source_dir(SOURCE_DIR)
+ep_get_binary_dir(BINARY_DIR)
+
+set(CFG_ENV_VAR "CC=\"${CC}\" CFLAGS=\"${CFLAGS}\" LDFLAGS=\"${LDFLAGS}\" ")
+set(CFG_OPTS "-q --prefix=${BINARY_DIR} --disable-static --enable-shared --host=\"${CHOST}\"")
```

Comment:
Hi, me and my crusade against quiet stuff would like to get rid of the quiet flag ;p.

Reply:
I agree, let's avoid quiet unless it causes some stupid "log size too big" or whatever issue on one of the CI platforms.